qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -388,7 +380,7 @@ def run(
             self,
             qobj: Qobj,
             backend_options: Optional[Dict] = None,
-            noise_model: Optional[Any] = None
+            noise_model=None
```

Comment:
There was no `NoiseModel` type in the repository, so I decided to leave it unmarked instead of specifying `Any`.